src/Model/helpers.js

```diff
@@ -21,3 +21,24 @@ export const createTimestampsFor = (model: Model) => {
 
   return timestamps
 }
+
+export async function fetchChildren(model: Model): Promise<Model[]> {
+  const { associations } = model.collection.modelClass
```

Comment:
this can be simplified -- `model.constructor` or something like that -- it's a javascript thing